### ISSUE TITLE
Update lodash to pass npm audit

### ISSUE THREAD

**Issue by kevinkace** [inikulin/publish-please#76]
I'm getting an error when trying to publish my package as the version of lodash in publish-please contains a vulnerability.

**Comment by hdorgeval** [comment]
Hi @kevinkace, which version of publish-please are you using?

**Comment by hdorgeval** [comment]
@kevinkace : thank you for your feedback.
 As a temporary fix, until I do some correction on my side, you have two possible workarounds:

1°) Create a `.auditignore` file at the root of your project and insert in it the following line:
```sh
https://npmjs.com/advisories/577
```

2°) Remove the publish-please dependency and scripts associated in the `package.json` file:

```scripts
"publish-please": "publish-please",
    "prepublish": "publish-please guard"
```

Keep the .publishrc file
then use `npx publish-please` to publish your package

**Comment by hdorgeval** [comment]
@kevinkace : if you choose the second option, do not forget to re-generate the `package-lock.json` file

**Comment by kevinkace** [comment]
@hdorgeval thanks, I'll give it a shot.

**Comment by hdorgeval** [comment]
@kevinkace : thank you for your feedback. 
I opened a PR to solve this issue: [Replace ban-sensitive-files dependency with a custom sensitive-data validation](https://github.com/inikulin/publish-please/pull/77)

**Comment by hdorgeval** [comment]
@kevinkace : I leave this issue opened until PR#77 is merged.

**Comment by hdorgeval** [comment]
Hi @kevinkace, this issue is solved in `publish-please@5.0.0`. Could you give it a try?
Any feedback is welcomed.

**Comment by kevinkace** [comment]
Success!! Thank you for the update.
